platform/platform.json

```diff
@@ -1,7 +1,7 @@
 {
-  "version": "1.2.2",
-  "mobileAppVersion": "1.2.2",
-  "releaseNotes": "* Wordpress support, performance improvements and bug fixes",
+  "version": "1.2.3",
```

Comment:
Why there aren't any updated versions of extensions? I'm sure that Youtube and WordPress have a newer version at least. Also, revert the change made in release notes. The idea that internal releases keep the release notes, so the final, public version has listed all the improvements we are publishing.